packages/tailwindcss/src/utilities.ts

```diff
@@ -1901,14 +1878,12 @@ export function createUtilities(theme: Theme) {
   suggest('grid-cols', () => [
     {
       values: Array.from({ length: 12 }, (_, i) => `${i + 1}`),
-      valueThemeKeys: ['--grid-template-columns'],
```

Comment:
We still need all of these theme keys for backwards compatibility, that's why these all exist right now 👍